ansible/roles_ocp_workloads/ocp4_workload_cloud_native_robot/templates/cicd/applicationset.yaml.j2

```diff
@@ -0,0 +1,44 @@
+---
+apiVersion: argoproj.io/v1alpha1
+kind: ApplicationSet
+metadata:
+  name: cicd
+  namespace: openshift-gitops
+spec:
+  generators:
+  - list:
+      elements:
+{% for n in range(1, ocp4_workload_cloud_native_robot_shared_gitea_user_count | int + 1 ) %}
+      - user: {{ ocp4_workload_cloud_native_robot_shared_gitea_user_prefix }}{{ n }}
```

Comment:
This appears to setup for multiple users but no `agnosticd_user_info` reports information for the users?

Reply:
Yes, this just configures the user namespaces via git ops approach. The main multiple users approach is working with the default demo.redhat.com approach & showroom. 

There is nothing to provide via agnosticd_user_info in that particular step.